### DISCUSSION TITLE
Welcome to DeepSentiPers Discussions!

### DISCUSSION THREAD

**Discussion by JoyeBright** [JoyeBright/DeepSentiPers#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by morhamz** [comment]
My name is Morteza Hamzei, I have a master's degree in data mining and I have 25 years of experience in writing systems for commercial systems. I work in text mining and data mining and I love research and learning. I will be extremely happy if your collection is dear.
